### ISSUE TITLE
Handle undocumented API changes

### ISSUE THREAD

**Issue by nsjames** [eosnetworkfoundation/devrel#205]
https://github.com/AntelopeIO/leap/issues/80

get_transaction_status: https://github.com/AntelopeIO/leap/commit/342edbdf9caf9578360847bbb7817da058b254e3
send_transaction2: https://github.com/AntelopeIO/leap/releases/tag/v3.1.0
compute_transaction: ovi has notes

AC:
- Update [swagger files](https://github.com/AntelopeIO/leap/blob/main/plugins/chain_api_plugin/chain.swagger.yaml) to include
  - get_transaction_status
  - compute_transaction
  - send_transaction2

**Comment by bhazzard** [comment]
Closed https://github.com/AntelopeIO/leap/issues/80 as duplicate of this issue.

**Comment by iamveritas** [comment]
requires feedback from @ericpassmore because I can't verify it locally, can't generate locally the docs with these changes.

**Comment by ericpassmore** [comment]
Updated [docsgen/ehp/how-toupdate-in-place](https://github.com/eosnetworkfoundation/docsgen/tree/ehp/how-to-update-in-place) clean_rebuild.sh to build leap 3.2 and 4.0. 

Currently hardcoded to a personal branch for Leap 4.0 release. Waiting on syntax fix for producer_plugin.swagger.xml before creating PR to merge into staging

**Comment by iamveritas** [comment]
PRs that covered this task:
https://github.com/AntelopeIO/leap/pull/1087
https://github.com/AntelopeIO/leap/pull/1110
